### DISCUSSION TITLE
HELP! Conversion of EE image collection to xarray

### DISCUSSION THREAD

**Discussion by eilishb25** [google/Xee#133] [General]
Hi there, I am attempting to use xarray.open_dataset for a pre-processed and clipped EE image collection as follows:

ds = xarray.open_dataset(marebone_im_coll1, engine='ee', crs='EPSG:28357', scale=10) -> scale representing 10m resolution

I applied crs='EPSG:28357' which is the projection I wish to use. However, this doesn't seem to work. When I apply the GDA EPSG:4283 projection, i get an xarray output wtih an odd number of lon/lat coordinates and im also not sure what the appropriate scale would be here? like is it the equivalent radians to achieve a 10m resolution?

Any help is appreaciated

**Comment by jdbcode** [comment]
This may have been fixed in #156.

@eilishb25 I wonder if you could upgrade your Xee install and try again and let us know.
